torch/_tensor.py

```diff
@@ -1011,6 +1011,33 @@ def rename(self, *names, **rename_map):
         # See Note [rename_ / rename API]
         return update_names(self, names, rename_map, inplace=False)
 
+    def to_sparse_coo(self):
+        """ Convert a tensor to coordinate format. Only works with 2D tensors.
```

Comment:
`to_sparse_coo` works with tensors with arbitrary dimensions. Hence:
```suggestion
        """ Convert a tensor to :ref:`coordinate format <sparse-coo-docs>`.
```

Reply:
Well CSR can only be 2D so it;s technically true :D

Reply:
Sure. Although this PR is about CSR, the method applies to tensors with all possible layouts.

Reply:
O yeah, true. I missed the context there.